waik/waik_provision_unattend.xml.erb

```diff
@@ -0,0 +1,158 @@
+            </ImageInstall>
+            <UserData>
+                <AcceptEula>true</AcceptEula>
+            </UserData>
+        </component>
+        <component name="Microsoft-Windows-International-Core-WinPE" processorArchitecture="amd64" publicKeyToken="31bf3856ad364e35" language="neutral" versionScope="nonSxS" xmlns:wcm="http://schemas.microsoft.com/WMIConfig/2002/State" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+            <SetupUILanguage>
+                <UILanguage>en-US</UILanguage>
+            </SetupUILanguage>
+            <% if @host.params['systemLocale'] -%>
+              <InputLocale><%= @host.params['systemLocale'] %></InputLocale>
+              <SystemLocale><%= @host.params['systemLocale'] %></SystemLocale>
+              <UILanguageFallback>en-US</UILanguageFallback>
+              <UILanguage><%= @host.params['systemUILanguage'] %></UILanguage>
+              <UserLocale><%= @host.params['systemLocale'] %></UserLocale>
+            <% else -%>
+               <InputLocale>en-US</InputLocale>
+              <SystemLocale>en-US</SystemLocale>
+              <UILanguageFallback>en-US</UILanguageFallback>
+              <UILanguage>en-US</UILanguage>
+              <UserLocale>en-US</UserLocale>
+            <% end -%>
+        </component>
+    </settings>
+    <settings pass="oobeSystem">
+        <component name="Microsoft-Windows-International-Core" processorArchitecture="amd64" publicKeyToken="31bf3856ad364e35" language="neutral" versionScope="nonSxS" xmlns:wcm="http://schemas.microsoft.com/WMIConfig/2002/State" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+            <% if @host.params['systemLocale'] -%>
+              <InputLocale><%= @host.params['systemLocale'] -%></InputLocale>
+              <SystemLocale><%= @host.params['systemLocale'] -%></SystemLocale>
+              <UILanguageFallback>en-US</UILanguageFallback>
+              <UILanguage><%= @host.params['systemUILanguage'] -%></UILanguage>
+              <UserLocale><%= @host.params['systemLocale'] -%></UserLocale>
+            <% else -%>
+               <InputLocale>en-US</InputLocale>
+              <SystemLocale>en-US</SystemLocale>
+              <UILanguageFallback>en-US</UILanguageFallback>
+              <UILanguage>en-US</UILanguage>
+              <UserLocale>en-US</UserLocale>
+            <% end -%>
+        </component>
+        <component name="Microsoft-Windows-Shell-Setup" processorArchitecture="amd64" publicKeyToken="31bf3856ad364e35" language="neutral" versionScope="nonSxS" xmlns:wcm="http://schemas.microsoft.com/WMIConfig/2002/State" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+            <UserAccounts>
+                <AdministratorPassword>
+                    <PlainText>false</PlainText>
+                    <Value><%= Base64.encode64(Encoding::Converter.new("UTF-8", "UTF-16LE",:undef => nil).convert(Base64.decode64(root_pass)+"AdministratorPassword")).delete!("\n").chomp %></Value>
```

Comment:
@domcleal Another idea:

We ship two templates for this; one working with safe mode, the other requiring it to be off. The one for safe mode would render `root_pass` directly. The user would need to append the string `AdministratorPassword` to the root password.

If `safe_mode?` is available at any point it could be merged again.

I still like the encoder better because it is more error prone (eg, the template will not render if password is encoded in a wrong way). 

Also, since Base64 password encoding is there, maybe there will be `Plain` in the future ;)